GameServer/Program.cs

```diff
@@ -1,63 +1,26 @@
-using LeagueSandbox.GameServer.Core.Logic;
-using LeagueSandbox.GameServer.Core.Logic.RAF;
-using LeagueSandbox.GameServer.Logic;
-using LeagueSandbox.GameServer.Logic.GameObjects;
-using LeagueSandbox.GameServer.Logic.Items;
 using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Linq;
-using System.Net;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using ENet;
+using Ninject;
+using LeagueSandbox.GameServer;
 
 namespace LeagueSandbox
 {
     class Program
     {
-        private static uint SERVER_HOST = Address.IPv4HostAny;
-        private static ushort SERVER_PORT = 5119;
-        private static string SERVER_KEY = "17BLOhi6KZsTtldTsizvHg==";
-        private static string SERVER_VERSION = "0.2.0";
+        // TODO: Require consumers of this inject a ServerContext
         public static string ExecutingDirectory;
```

Comment:
Change this to the C# get/set format as well while you're at it: `public static string ExecutingDirectory { get; private set; }`